app/controllers/api/v1/wb_extra_country_data_controller.rb

```diff
@@ -0,0 +1,60 @@
+module Api
+  module V1
+    class WbExtraCountryDataController < ApiController
+      def show
+        country = Location.find_by(location_type: 'COUNTRY',
+                                   iso_code3: params[:iso])
+        unless country
+          render json: {
+            error: 'Country not found',
+            status: 404
+          }, status: :not_found and return
+        end
+
+        render json: filtered_country_data(country),
+               each_serializer: Api::V1::WbExtra::CountryDataSerializer
+      end
+
+      private
+
+      def filtered_country_data(country)
```

Comment:
These filter methods are search helpers, and as such they belong in the model, not in the controller